frontend/apps/reader/modules/readertoc.lua

```diff
@@ -1147,6 +1187,17 @@ Enabling this option will restrict display to the chapter titles of progress bar
             UIManager:show(items_font)
         end,
     }
+    menu_items.toc_items_show_chapter_length = {
+        text = _("Append chapter length"),
```

Comment:
Thank you for this enhancement!

>Is "Append" too technical, and should we just use "Show chapter length" ?

Yes, I think the simple "show" is clearer in this context. Also, maybe a bit more to the point, "Show chapter page counts"? Either way, it should be plural (lengths/counts), because it's for all chapters.

Reply:
("Show chapter length" (singular) also reads/sounds better to me learning English :))

Reply:
I changed my mind, i like the first one better. From a user point of view that is.

Reply:
> (I also learned the singular/plural pecularities with @Frenzie.)

They're not always the same across all varieties. But if you want plural you'd avoid the discussion with `Show page count for chapters` since that can't be chapter.

Reply:
Any consensus ?
(`Show page count for chapters` feels twisted to me, and would imply there are _other things than chapters_, for which we won't show any _page count_).

Reply:
I'm not opposed to append. It's not just technical, even if it is a bit formal.

Reply:
No strong preference-

- Show chapter lengths
- Show chapter length
- Show chapter page counts
- Show chapter page count

are all okay with me. "Append" is also okay but it's a bit too formal for non-technical usage and feels a bit too 'permanent' to me; not something that can easily be toggled on and off.